ts/components/wallet/SelectionBox.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { TouchableOpacity, View, StyleSheet } from "react-native";
+import { H4 } from "../core/typography/H4";
+import { LabelSmall } from "../core/typography/LabelSmall";
+import { IOColors } from "../core/variables/IOColors";
+
+type Props = {
+  logo?: React.ReactNode;
+  mainText: string;
+  subText: string;
+  ctaText?: string;
+  onPress?: () => void;
+  accessibilityLabel?: string;
+};
+
+const styles = StyleSheet.create({
+  selectionBox: {
+    borderWidth: 1,
+    borderColor: IOColors.bluegreyLight,
+    borderRadius: 8,
+    padding: 16,
+    flexDirection: "row",
+    alignItems: "center"
+  },
+
+  selectionBoxIcon: {
+    flexGrow: 0,
+    flexShrink: 0,
+    flexBasis: "auto",
+    paddingRight: 24
+  },
+
+  selectionBoxContent: {
+    flexGrow: 1,
+    flexShrink: 1,
+    flexBasis: "100%"
+  },
+
+  selectionBoxTrail: {
+    flexGrow: 0,
+    flexShrink: 0,
+    flexBasis: "auto",
+    paddingLeft: 24
+  }
+});
+
+/**
+ * A bordered and rounded box made up by three columns:
+ * - The first one containing an icon
+ * - The second one with a title and a subtitle
+ * - The third one containing a CTA
+ */
+export const SelectionBox = (props: Props) => (
+  <TouchableOpacity
```

Comment:
```suggestion
  <TouchableDefaultOpacity
```
I don't know if this design guide line is still valid, but we use it widely in the UI building
cc @thisisjp 

### `TouchableOpacity`
https://user-images.githubusercontent.com/822471/159759666-f872541e-3fa9-4198-aa0c-0c988a2742fb.mov

### `TouchableDefaultOpacity`
https://user-images.githubusercontent.com/822471/159759712-78bf479d-b7e6-4678-97f8-bc503e956902.mov

Reply:
@Undermaken valid from a UX side, it could be improved on a visual side in the future :D

Reply:
Gotcha, I completely missed that component, switched here: https://github.com/pagopa/io-app/pull/3840/commits/f6a16e28c605772a0be120592eeb190d39aa3e94 👍